### ISSUE TITLE
Stepper step text label contras issue in Blau

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2605]
## Description

https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcIA8BlALjADpmAnAOgHYAExYArrrjIWgJKGwAeAvMAEwC%2BRpAzhpl5sA2vhAAhAIa8AlmGIARGGkkyANrzEAaYmIDCEALaZJhAJ6Llqjdt0gU5AEa8wuGZjQyIJAAprTYgC63IQA9AB8RCBaIGgAFjCGMLwIwhL%2B5CCBMQDuMlDxKfDCAMwAbAAMwUA

Current step label is failing contrast against step background, it should use textPrimaryNegative

## Tasks

- [x] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2423)
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-17603)
